tests/spec/fs.link.spec.js

```diff
@@ -111,3 +111,119 @@ describe('fs.link', function() {
     });
   });
 });
+
+
+
+
+
+describe('fs.promises.link', function() {
+  beforeEach(util.setup);
+  afterEach(util.cleanup);
+
+  it('should be a function', function() {
+    var fs = util.fs();
```

Comment:
You can use the promise API directly with:
`var fs = util.fs().promises;`

Reply:
I tried checking out other implementations of this so I could see how I could do this but other people's code did not do this. By chance do you mind giving me more information about how to implement it?

Reply:
I think it's fine either way.  The `util.fs().promises` way would be good if you were needing to use it multiple times, and you could cache it like this.  In this case, it's just once, so either or is fine.